src/components/Statistics.tsx

```diff
@@ -0,0 +1,158 @@
+  const [blocks, setBlocks] = useState('')
+  const [avgBlockTime, setAvgBlockTime] = useState('')
+  const [isLoading, setIsLoading] = useState(true)
+
+  const currentSupplyPercentage = ((parseFloat(circulating) / parseFloat(totalSupply)) * 100).toPrecision(3)
+  const hashrateInteger = thousandsSeparation(parseInt(hashrate))
+  const hashrateDecimal = parseFloat(hashrate).toFixed(2).split('.')[1]
+
+  useEffect(() => {
+    Promise.all([
+      client?.charts
+        .getChartsHashrates({
+          fromTs: new Date().getTime() - ONE_DAY,
+          toTs: new Date().getTime(),
+          'interval-type': 'hourly'
+        })
+        .then((res) => res.data)
+        .then((values) => setHashrate(values[0]?.value || '-')),
+      client?.infos
+        .getInfosHeights()
+        .then((res) => res.data)
+        .then((values: PerChainValue[]) =>
+          setBlocks(values.reduce((acc: number, { value }) => acc + value, 0).toString())
+        ),
+      client?.infos
+        .getInfosSupplyTotalAlph()
+        .then((res) => res.text())
+        .then((value) => setTotalSupply(parseInt(value).toString())),
+      client?.infos
+        .getInfosSupplyCirculatingAlph()
+        .then((res) => res.text())
+        .then((value) => setCirculating(parseInt(value).toString())),
+      client?.infos
+        .getInfosTotalTransactions()
+        .then((res) => res.text())
+        .then((value) => setTransactions(value)),
+      client?.infos
+        .getInfosAverageBlockTimes()
+        .then((res) => res.data)
+        .then((values) =>
+          setAvgBlockTime(
+            abbreviateValue(values.reduce((acc: number, { value }) => acc + value, 0.0) / values.length, SUFFICES_TIME)
+          )
+        )
+    ]).then(() => setIsLoading(false))
+  }, [refresh, client])
+
+  return (
+    <Block>
+      <StatisticBlock
+        title={'Hashrate'}
+        primary={
+          <>
+            {hashrate !== '-' && (
+              <span>
+                {hashrateInteger}.<TextSecondary>{hashrateDecimal}</TextSecondary>
+              </span>
+            )}
+            {hashrate === '-' && <span>-</span>}
+          </>
+        }
+        secondary={<span>GH/s</span>}
+        isLoading={isLoading}
+      />
+      <StatisticBlock
+        title={'Supply'}
+        primary={
+          <>
+            <span>{abbreviateValue(parseInt(circulating), SUFFICES_QUANTITY)}</span>
+            <TextSecondary> / </TextSecondary>
+            <TextSecondary>{abbreviateValue(parseInt(totalSupply), SUFFICES_QUANTITY)}</TextSecondary>
+          </>
+        }
+        secondary={
+          <span>
+            <TextPrimary>{currentSupplyPercentage}%</TextPrimary> is circulating
+          </span>
+        }
+        isLoading={isLoading}
+      />
+      <StatisticBlock
+        title={'Transactions'}
+        primary={abbreviateValue(parseInt(transactions), SUFFICES_QUANTITY)}
+        secondary={<div>Total</div>}
```

Comment:
Are these `div`s really needed? Their parent seems to be just another `div`. Would it not work well if it was just `secondary="Total"`? (also below)